### PR TITLE
User/nathanaelhuh/sft 50 add header

### PR DESCRIPTION
**Pull request by nathanaelhuh**

### CODE REVIEW

File: src/app/app.component.ts [hunk 1]
```diff
@@ -1,10 +1,73 @@
 import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+// import {Animate } from '@angular/animations'
```
Comment (liconway) [inline src/app/app.component.ts]: Is this (going to be) needed?

File: src/app/app.component.ts [hunk 2]
```diff
@@ -1,10 +1,73 @@
...
+    let tabsclass = 'mat-tab-link';
+    if (this.activeLinkIndex === indexofRouteLink) {
+      tabsclass = 'mat-tab-link mat-tab-label-active';
+    }
+    return tabsclass;
+  }
+
+  toggle(): void {
+    this.isOpen = !this.isOpen;
+    console.log("OPEN");
```
Comment (liconway) [inline src/app/app.component.ts]: Assuming this was meant for debugging

File: src/app/app.component.ts [hunk 3]
```diff
@@ -1,10 +1,73 @@
...
+    // x.classList.toggle("change");
```
Comment (bill-luu) [inline src/app/app.component.ts]: Dead comment?

File: src/app/app.module.ts [hunk 4]
```diff
@@ -1,5 +1,10 @@
 import { BrowserModule } from '@angular/platform-browser';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NgModule } from '@angular/core';
+import { MatTabsModule } from '@angular/material/tabs';
+import {MatButtonModule} from '@angular/material/button';
+import {MatButtonToggleModule} from '@angular/material/button-toggle';
+import {MatMenuModule} from '@angular/material/menu';
```
Comment (bill-luu) [inline src/app/app.module.ts]: Minor, but the spacing of the import is inconsistent.
`import {MatButtonModule}` -> `import { MatButtonModule }`

**Comment by bill-luu** [comment]
I also got a failed to compile :)
`ERROR in src/app/app.module.ts:33:5 - error TS1005: ',' expected.`

File: src/app/app.module.ts [hunk 5]
```diff
@@ -9,19 +14,36 @@ import { InKindDonationsComponent } from './components/support-us-page/in-kind-d
...
+
 @NgModule({
   declarations: [
     AppComponent,
+    CarsComponent,
+    SupportUsComponent,
+    SponsorsComponent,
+    NewsComponent,
+    ContactUsComponent,
+    TeamComponent
```
Comment (JaydenBischoff) [inline src/app/app.module.ts]: Missing a comma after TeamComponent.

File: src/app/app-routing.module.ts [hunk 6]
```diff
@@ -1,11 +1,27 @@
...
+	{ path: 'cars', component: CarsComponent },
+    { path: 'support-us', component: SupportUsComponent },
+    { path: 'sponsors', component: SponsorsComponent },
+    { path: 'news', component: NewsComponent },
+    { path: 'team', component: TeamComponent },
+    { path: 'contact-us', component: ContactUsComponent },
+    { path: '**', redirectTo: '/', pathMatch: 'full' }
+    ];
 
+// const routes: Routes = [
```
[the remark this answers is not in the document]
Comment (nathanaelhuh) [reply src/app/app-routing.module.ts]: Nope! Will delete it

**Pull request merged**